docs/Rootless.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Rootless
+layout: docs
+---
+
+### Context
+
+Practically all jailbreaks prior to iOS 15 have been 'rootful' as they work on and install files to the (root) system directories (e.g., `/usr`, `/Library`, `/Applications`, etc). Moving forward with iOS 15 and beyond, most if not all jailbreaks will have to be 'rootless' as Apple now prevents writing to the system directories with a protection called [Signed System Volume (SSV)](https://support.apple.com/guide/security/signed-system-volume-security-secd698747c9/web). There are workarounds for this, namely bindFS, but these are less than ideal.
+
+To work around SSV, the [Procursus Team](https://github.com/procursusteam/) has thought up a rootless scheme that is now the de facto rootless standard. The changes namely involve working in and installing tweaks/addons to `/var/jb` and using the `iphoneos-arm64` package architecture. To learn more about the specifics of this implementation, see [The Apple Wiki](https://theapplewiki.com/wiki/Rootless).
+
+With this new target location, *all* projects will have to be recompiled to reference resources from and install to `/var/jb`. This involves changing the internal structure of .deb's to include this new path and specifying library and framework install paths using `@rpath` instead of absolute paths. Theos will handle both of these changes for you. For more information on what `@rpath` is and how it works, see Mike Ash's [informative blog post](http://www.mikeash.com/pyblog/friday-qa-2009-11-06-linking-and-install-names.html).
+
+### Implementation
+
+---
+
+**Important Notes**:
+- You must run `make clean` when switching between a rootful and rootless build
```

Comment:
I think this is more of a best practice, but it is required to build correctly if your build targets are different for rootful and rootless for example.

Reply:
I believe there was an issue with the path macros defined in rootless.h retaining the package install prefix between builds unless the build was cleaned (cc @NighwindDev).

Reply:
Ah yes, that didn't seem to happen to me for some reason but I remember that issue now.